File: java/link-message-client/src/main/java/link/message/client/content/complex/linear/FormFieldValue.java
package link.message.client.content.complex.linear;

/**
 * 表单字段对应的值信息
 * 
 * @author zhongt
 *
 */
public class FormFieldValue {
	protected String key;
	protected String value;
	
	public FormFieldValue(String key, String value) {
		this.key   = key;
		this.value = value;
	}

	public String getKey() {
		return key;
	}

	public void setKey(String key) {
		this.key = key;
	}

	public String getValue() {
		return value;
	}

	public void setValue(String value) {
		this.value = value;
	}
}

File: net/Link.Message.Client/content/complex/linear/FormFieldValue.cs
namespace Link.Message.Client.content.complex.linear
{

	/// <summary>
    /// 表单字段对应的值信息
	/// </summary>
	public class FormFieldValue
	{
		protected internal string key;
		protected internal string value;

		public FormFieldValue(string key, string value)
		{
			this.key = key;
			this.value = value;
		}

		public virtual string Key
		{
			get
			{
				return key;
			}
			set
			{
				key = value;
			}
		}


		public virtual string Value
		{
			get
			{
				return value;
			}
			set
			{
				this.value = value;
			}
		}

	}
}